docs/releasing.md

```diff
@@ -7,7 +7,50 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-### Releasing KubeFed
+## Releasing KubeFed
+
+### Automated
+
+Creating a KubeFed release can be automated by running a couple of scripts.
+
+##### Prerequisites
+
+The scripts used to automated the release of KubeFed rely on the following
+prereqs:
+
+- [travis CLI](https://github.com/travis-ci/travis.rb#installation)
+    - Once installed verify you can run `travis branches --repo
+      kubernetes-sigs/kubefed` without prompting for interaction.
+- [hub CLI](https://github.com/github/hub#installation)
```

Comment:
Would it make sense to automate the installation of the hub cli?

Reply:
I think it's preferable to automate the dependency installation rather than documenting it (since automation is effectively docs for a developer). I'm fine with deferring it to a subsequent PR, but I think automating dependency installation should be a near-term priority. Adding it to `download-binaries.sh` would be fine - I don't think it's necessary to create a separate script just for release dependencies.